package.json

```diff
@@ -69,7 +69,7 @@
     "tslint": "^5.10.0",
     "tslint-config-standard": "^7.1.0",
     "typescript": "^2.9.2",
-    "wechaty": "^0.17.133"
+    "wechaty": "^0.19.107"
```

Comment:
I have to notify you that this is a BREAKING change and you should bump your MINOR version number when you **changing the dependence to wechaty 0.17 to 0.19**